### PR TITLE
Feature run endpoint

### PR DESCRIPTION
**Pull request by AlexPrapugicu**

### CODE REVIEW

File: rpi/boundary/car_controller.py [hunk 1]
```diff
@@ -3,20 +3,25 @@
...
 
 from car.car import Car
+from service.image_analysis import ImageAnalysisService
 
 
 class CarController:
-    def __init__(self, car: Car, flask_app: Flask):
+    def __init__(self, car: Car, flask_app: Flask, image_analysis_service: ImageAnalysisService):
         self._car = car
         self._flaskApp = flask_app
```
Comment (pflueras) [inline rpi/boundary/car_controller.py:12]: Use camel case for attributes: e.g. `self._imageAnalysisService`

File: rpi/service/image_analysis.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import time
+
+
+class ImageAnalysisService:
+
```
Comment (pflueras) [inline rpi/service/image_analysis.py:5]:
```suggestion
def upload_image(self, image_filename: str)
```

File: rpi/service/image_analysis.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    def uploadImage(self, image_filename: str):
+        time.sleep(0.25)
+
```
Comment (pflueras) [inline rpi/service/image_analysis.py:8]:
```suggestion
def detect_traffic_light(self, image_filename) -> bool:
```

**Comment by AlexPrapugicu** [comment]
Solved some conflicts and renamed variables according to requirements.

**Pull request merged**